Recurly/Resources/Coupon.cs

```diff
@@ -80,10 +80,6 @@ public class Coupon : Resource {
     [JsonProperty("name")]
     public string Name { get; set; }
   
-    /// <value>Plans</value>
-    [JsonProperty("plans")]
-    public List<PlanMini> Plans { get; set; }
```

Comment:
Are we going to add this back in when 2018-10-04 is GA?

Reply:
Nevermind, I just realized this is a generated file and I see exactly how it will be.